workflow-fingerprint-collector/src/main/java/ca/on/oicr/pde/workflows/FingerprintCollectorWorkflow.java

```diff
@@ -34,11 +23,10 @@
     private String[] bamFiles;
     private String[] vcfFiles;
     private String[] gatkDirs;
-    private String studyName = "";
     private String dataDir;
-    private final String tempDir = "tempfiles/";
-    private final String gatkTmp = "temp";
-    private final String finDir  = "finfiles/";
+    private final static String tempDir = "tempfiles/";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)